tests/schemas/dump04_default.esdl

```diff
@@ -23,3 +23,31 @@ type Test1 {
         default := <array<tuple<name: str, severity: int16>>>[]
     };
 };
+
+type Test2 {
```

Comment:
Can you also add some globals + some schema acl policies and test that they all get restored and can be introspected/verified after restore?